packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -97,7 +98,10 @@ const RealAccountSignup = ({
                     onError={showErrorModal}
                 />
             ),
-            title: () => localize('Add or manage account'),
+            title: local_props =>
+                local_props.has_fiat && local_props.available_crypto_currencies?.length === 0
+                    ? localize('Manage account')
+                    : localize('Add or manage account'),
         },
         {
             body: local_props => (
```

Comment:
![image](https://user-images.githubusercontent.com/56330681/119661282-c1e98d00-be4d-11eb-9225-ee644994dab6.png)
I think we should change the title in here too